added disclaimer about support and partner portal (#1322)
spryker/spryker-docs: docs/scos/user/intro-to-spryker/support/how-to-use-the-support-portal.md

{% info_block infoBox "Info" %}

Allow 3-5 days of processing time for change requests.
If you need changes done at a certain date, include that information in the change request description.

> Please note that only customers (via the support portal) can request new environments or access to environment monitoring. For contractual reasons it is not possible to request these items by partners.

Reviewer: ```suggestion
Also, note that only customers (through the support portal) can request new environments or access to environment monitoring. For contractual reasons, partners can't request these items.
```
